src/App.module.css

```diff
@@ -88,16 +88,6 @@
   gap: 20px;
 }
 
-.headerRight {
-  height: 100%;
-  gap: 10px;
-}
-
-.copyUrlButton {
```

Comment:
I know these are deleted, and it doesn't really matter, but css modules allows you to name your classes `copy-url-button` and it will convert it to camel case so your css still looks like css